tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -185,8 +189,16 @@ static struct bt_gatt_cb gatt_callbacks = {
 	.att_mtu_updated = att_mtu_updated,
 };
 
-static void init(void)
+static void cap_disconnected_cb(struct bt_conn *conn, uint8_t reason)
+{
+	k_sem_give(&sem_disconnected);
+}
+
+static void init(size_t acceptor_cnt)
 {
+	static struct bt_conn_cb conn_cb = {
```

Comment:
That's a fair point. 

I'd rather not move other code to keep the changes to minimum. 
Declaring it here like this can be considered the recommended way to reduce scope of the variable. 

Suggest to move the other parts after this PR if we really want to do that